### DISCUSSION TITLE
Model

### DISCUSSION THREAD

**Discussion by guyzilb** [vanna-ai/vanna#302] [Q&A]
vanna_model_name = 'pce'
vn = VannaDefault(model=vanna_model_name, api_key=****)

Is there any way to create "vanna_model_name" through Python code and not just from UI ?

**Comment by zainhoda** [comment]
@guyzilb could you elaborate on your use case? We removed this functionality in order to keep the behavior consistent across the various configuration options. We might potentially add this functionality back in a future release but I'd like to understand what your goal is in order to help us decide where it should go

**Reply by guyzilb** [reply]
let's say I want to create a separate flask app 
and they completely separated, working on different databases with different data, if I use the same model the model will train for all the data and it will be not correct for me when someone asks Q 
so I need to generate a model automatically every time I run a new app for a client.

**Reply by zainhoda** [reply]
Ah yeah, I got it -- we'll add this back as part of a set of "extra" functionality outside of the base class

**Comment by guyzilb** [comment]
can you share here please how to use it?

**Reply by guyzilb** [reply]
@zainhoda  ?

**Reply by zainhoda** [reply]
@guyzilb can you email me at zain@vanna.ai from the email that you used to sign up for Vanna? We are activating this on a case by case basis
